lib/core-common/src/utils/__tests__/template.test.ts

```diff
@@ -11,7 +11,7 @@ describe('server.getPreviewHeadHtml', () => {
   describe('when .storybook/preview-head.html does not exist', () => {
     beforeEach(() => {
       mock({
-        [`${__dirname}/../../templates/base-preview-head.html`]: BASE_HTML_CONTENTS,
```

Comment:
So interesting... The FS mocking library makes `pkg-dir` `sync` command return `undefined`...

It's hacky, I know... But it does work...